webapp/tests/karma/ts/test-utils/test-setup.ts

```diff
@@ -0,0 +1,42 @@
+import { TestBed, TestModuleMetadata } from '@angular/core/testing';
+import { MaterialTestModule } from './material-test.module';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+
+/**
+ * Helper function to configure testing modules with proper Material icon support
+ * to prevent the ":icon-back" error in unit tests
+ */
+export const configureTestingModule = (config: TestModuleMetadata): void => {
+  // Ensure imports array exists
+  config.imports = config.imports || [];
+
+  // Add HttpClientTestingModule for tests that need HTTP
```

Comment:
I think we should not add this.
We would often see this when a dependency chain is altered, and requests end up fired. This error should blow up the test and make us diligent of mocking dependencies correctly in tests.